gluster-block.spec.in

```diff
@@ -27,6 +27,7 @@ BuildRequires:    systemd-units
 
 Requires:         tcmu-runner >= 1.0.4
 Requires:         targetcli >= 2.1.fb43
+Requires:         iscsi-initiator-utils
```

Comment:
This change is not needed, as iscsid.service is only needed by the initiator, that will be started by multipathd.service as a dependency.

The issue that you have encountered before is getting fixed in https://review.gluster.org/#/c/17604/ which drops iscsid.service.